import_obfuscation.go

```diff
@@ -44,34 +47,55 @@ type privateImports struct {
 	privateNames []string
 }
 
-func obfuscateImports(objPath, importCfgPath string) (map[string]string, error) {
+func appendPrivateNameMap(nameMap map[string]string, packageDirectory string) error {
+	file, err := os.Open(filepath.Join(packageDirectory, garbleMapFile))
+	if errors.Is(err, os.ErrNotExist) {
```

Comment:
Why is this special, shouldn't this also error?

Reply:
This is the processing of a rare case when there are no private names in the package. In this case the map file is not created, but this is not an error.